Aliucord/src/main/java/com/aliucord/coreplugins/RemoveBilling.kt

```diff
@@ -0,0 +1,88 @@
+package com.aliucord.coreplugins
+
+import android.content.Context
+import android.view.*
+import android.widget.*
+import android.widget.TextView
+import androidx.core.content.ContextCompat
+import com.aliucord.Utils
+import com.aliucord.entities.CorePlugin
+import com.aliucord.patcher.*
+import com.lytefast.flexinput.R
+import com.discord.widgets.settings.WidgetSettings
+import com.lytefast.flexinput.fragment.FlexInputFragment
+import com.lytefast.flexinput.fragment.`FlexInputFragment$d`
+import com.discord.widgets.servers.guildboost.WidgetGuildBoost
+import com.discord.widgets.settings.profile.WidgetEditProfileBannerSheet
+import b.a.a.b.d
+import b.a.a.b.c
+import com.discord.widgets.stickers.UnsendableStickerPremiumUpsellDialog
+import com.discord.widgets.settings.WidgetSettingsMedia
+
+internal class RemoveBilling : CorePlugin(Manifest("RemoveBilling")) {
+    override val isHidden = true
+    
+    init {
+        manifest.description = "Removes all references to billing"
+    }
+
+    override fun start(context: Context) {
+        // Remove Billing settings
+        patcher.after<WidgetSettings>("onViewBound", View::class.java) { (_, view: View) ->
+            var container: LinearLayout = view.findViewById(Utils.getResId("nitro_settings_container", "id"))
```

Comment:
```suggestion
            var container = view.findViewById<LinearLayout>(Utils.getResId("nitro_settings_container", "id"))
```
I avoid defining the variable type rather than passing it directly to the reified function, it's unclear.
Also do this for any other instances of this happening in your changes.